### DISCUSSION TITLE
discussion/doc/test

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/discussion#7] [Announcements]
# discussion/doc/test

https://arxiv.github.io/discussion/doc/test.html

**Comment by cbf66** [comment]
This is our first comment of the new system.

*OK. Definitely have to change the theming to match the rest of the page*

**Reply by cbf66** [reply]
Reply to myself. Does it get indented?

**Reply by SBBCornell** [reply]
this is a new reply

**Reply by SBBCornell** [reply]
test test test *test* [whatever](whatever.com)

**Reply by DavidLFielding** [reply]
Test Reply

**Reply by cbf66** [reply]
Replies go at the bottom, right?

**Comment by mnazzaro** [comment]
Comment comment comment

**Reply by bmaltzan** [reply]
test

**Reply by SBBCornell** [reply]
Testing a comment

**Reply by bmaltzan** [reply]
- test3

**Comment by DavidLFielding** [comment]
Test comment!
